packages/website/dummy-data/typescript-project/webpack.config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  mode: 'development',
+  resolve: {
+    // Add `.tsx` as a resolvable extension.
+    extensions: ['.tsx', '.js'],
+  },
+  module: {
+    rules: [
+      { test: /\.tsx?$/, loader: 'babel-loader!ts-loader' },
```

Comment:
Pls can we use `@babel/preset-typescript`, or make an example with it.

Reply:
I think i'll leave this to test that we can work with projects using the actual typescript server. We have a custom babel test which should cover the babel-typescript use case